packetbeat/packetbeat.template-es2x.json

```diff
@@ -37,59 +40,79 @@
               "properties": {}
             },
             "auto-delete": {
+              "doc_values": true,
+              "index": true,
               "type": "boolean"
             },
             "class-id": {
+              "doc_values": true,
+              "index": true,
               "type": "long"
             },
             "consumer-count": {
+              "doc_values": true,
+              "index": true,
               "type": "long"
             },
             "consumer-tag": {
+              "doc_values": true,
               "ignore_above": 1024,
               "index": "not_analyzed",
               "type": "string"
             },
             "content-encoding": {
+              "doc_values": true,
               "ignore_above": 1024,
               "index": "not_analyzed",
               "type": "string"
             },
             "content-type": {
+              "doc_values": true,
```

Comment:
Since we almost never want to disable "doc_values", I wonder if it's worth adding it explicitly to the templates.